ext/hash/hash.c

```diff
@@ -1197,7 +1203,7 @@ PHP_FUNCTION(mhash)
 	char *data, *key = NULL;
 	size_t data_len, key_len = 0;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "ls|s!", &algorithm, &data, &data_len, &key, &key_len) == FAILURE) {
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "ls|s", &algorithm, &data, &data_len, &key, &key_len) == FAILURE) {
```

Comment:
```suggestion
	if (zend_parse_parameters(ZEND_NUM_ARGS(), "ls|s!", &algorithm, &data, &data_len, &key, &key_len) == FAILURE) {
```
Looks like this `!` got dropped by accident.

Reply:
Indeed. Thanks for checking through once more, will address this before merging.

Reply:
Oh, i didn't realize it could be pulled right away :) Just did it on both suggestions. Github makes it easy.

Thanks!